war/pom.xml

```diff
@@ -42,7 +42,7 @@ THE SOFTWARE.
     <contextPath>/jenkins</contextPath><!-- context path during test -->
     <port>8080</port><!-- HTTP listener port -->
     <node.version>4.0.0</node.version>
-    <npm.version>2.13.1</npm.version>
+    <npm.version>3.8.9</npm.version>
```

Comment:
We were using very old version. I found [this problem](https://github.com/mcasimir/mobile-angular-ui/issues/226#issuecomment-132317517).